test_pull_requests

```diff
@@ -458,6 +460,22 @@ module Hub
       comment += extended_tests if extended_tests
       recreate_comment(pull_id, bot_comment['id'], repo, comment)
 
+      bot_sha_comment = get_comment_with_prefix(pull_id, repo, BASE_SHA_PREFIX)
+      base_sha_command = %{
```

Comment:
Actually, thinking about it a bit more, the PR stores the commit ID of the base repo at the time the PR was created, but the local merge for the test is the *most recent* commit, which might not be the same as what the PR has.

So I was thinking instead of trying to calculate the base commit ID using `merge-base` during `--mark_test_success`, we could store it in the local repo's git config during `--local_merge_pull_request` by doing something like
`git config --local --add test-pull-requests.base-commit-id $(git rev-parse HEAD)`
before merging in the PR branch, right here: https://github.com/dobbymoodge/test-pull-requests/blob/50e86f0cb8218173c40d9b91c9945249e11764e6/test_pull_requests#L573